### ISSUE TITLE
MD075 false positive for prose containing multiple pipes after a table

### ISSUE THREAD

**Issue by solanab** [rvben/rumdl#818]
### rumdl version

0.2.55

### Installation method

Other (`pnpm dlx rumdl@0.2.55`)

### Bug description

MD075 treats a normal prose line containing two pipe characters as an orphaned table row when the line follows a valid GFM table separated by a blank line.

The same prose line is not reported when it appears without the preceding table, so the diagnostic depends on the table context rather than the line being a valid table row.

### Steps to reproduce

1. Create `test.md`:

```markdown
# Test

| A | B |
| - | - |
| x | y |

Noise low|med|high.
```

2. Run:

```console
rumdl check --no-config --no-cache test.md
```

For comparison, this standalone document produces no issue:

```markdown
# Test

Noise low|med|high.
```

### Expected behavior

No MD075 diagnostic. The blank line ends the table, and `Noise low|med|high.` is ordinary prose rather than an orphaned table row.

### Actual behavior

```text
test.md:7:1: [MD075] Orphaned table row(s) separated from preceding table by 1 blank line(s)

Found 1 issue(s) in test.md
```

### Configuration

```toml
# None; reproduced with --no-config and --no-cache.
```

### Operating system

Linux (`aarch64`, kernel `6.17.0-1010-oracle`)

### Additional context

This appears structurally related to #804 / #805, where prose after a table was misclassified because an Obsidian wikilink contained a pipe. That fix was intentionally scoped to the Obsidian flavor and does not cover ordinary prose with raw pipes under the standard flavor.

The issue is also related in principle to #557, which exempted pipe characters inside inline math from MD075 candidate-row detection.

**Comment by rvben** [comment]
Fixed in v0.2.56 (66d8d12c). A candidate orphan row after a table now has to carry an outer pipe, the way the table's own rows do, so `Noise low|med|high.` is prose again while a real orphaned `| x | y |` row still reports. Thanks for the minimal reproduction and the standalone control.
